engine/src/stack3.cpp

```diff
@@ -136,6 +136,12 @@ IO_stat MCStack::extendedload(MCObjectInputStream& p_stream, uint32_t p_version,
 	return t_stat;
 }
 
+IO_stat MCStack::load(IO_handle stream, uint32_t version)
+{
+	MCLog("%s should never be called!", __PRETTY_FUNCTION__);
+	MCUnreachable();
+}
+
 IO_stat MCStack::load(IO_handle stream, uint32_t version, uint1 type)
```

Comment:
Yes. However, it would still be a programming error to call `MCObject::load()` for an `MCStack` instance, so `load()` still needs to be overridden as above.  This is an interim fix.